freud/order.pyx

```diff
@@ -82,6 +82,11 @@ cdef class CubaticOrderParameter:
             orientation.
     """  # noqa: E501
     cdef freud._order.CubaticOrderParameter * thisptr
+    cdef t_initial
+    cdef t_final
+    cdef scale
+    cdef n_replicates
+    cdef seed
```

Comment:
Why do we need these? We already have properties for all of these that pull directly from the C++ class, is there a reason not to just use those?

Reply:
We would need to add properties for the `seed` and `n_replicates` as well; if you prefer we could eliminate the properties entirely and just make these attributes public. However, since we've in general avoided making `cdef`'ed attributes public, we'd probably have to introduce properties again to access the values, so it seems like a change that gains us nothing.

Reply:
That's fine for now, we can do a more thorough revision at some point.